### PR TITLE
move jquery-datetimepicker to npm/webpack

### PR DESCRIPTION
**Pull request by silverwind**
- update to latest version and move to npm
- adapt for api changes and css class rename
- add specificity to arc-green rules as dependency css now loads later
- use imports-loader to make it load correctly
- fix some wrong paths in librejs

### CODE REVIEW

**Comment by silverwind** [comment]
Resulting bundle is 335kB currrently because of the [dependencies](https://github.com/xdan/datetimepicker/blob/master/package.json#L34) this plugin has. Not sure if it's possible to prevent that second jQuery from loading.

It's only used on a single page (create milestone) and I think we should just use Fomantic's [Calendar](https://fomantic-ui.com/modules/calendar.html) to replace it entirely.

File: web_src/js/index.js [hunk 1]
```diff
@@ -731,15 +732,16 @@ function initRepository() {
...
       inline: true,
       timepicker: false,
       startDate: $datepicker.data('start-date'),
-      formatDate: 'Y-m-d',
-      onSelectDate(ct) {
-        $('#deadline').val(ct.dateFormat('Y-m-d'));
-      }
+      onSelectDate(date) {
+        $('#deadline').val(date.toISOString().substring(0, 10));
+      },
```
Comment (silverwind) [inline web_src/js/index.js:744]: I don't know why but the callback argument changed to return just a `Date`, so I had to adopt this.

**Comment by 6543** [comment]
@silverwind how hard is it to switch to "Fomantic's Calendar" directly instead?

**Comment by silverwind** [comment]
Probably not too hard but I've never worked with Fomantic stuff yet.

**Comment by silverwind** [comment]
Another issue with using `Calendar` is that we can not lazy-load Fomantic components (yet), so it would increase the size of the already big main Fomantic bundle as we're not using that component anywhere yet.

**Review by 6543 — APPROVED** [review]

**Review by gary-kim — APPROVED** [review]
LGTM

**Comment by techknowlogick** [comment]
ping LG-TM

**Pull request merged**